website/server/libs/config.js

```diff
@@ -0,0 +1,18 @@
+import nconf from 'nconf';
+
+const CHAT_APPROVAL_REQUIRED = nconf.get('CHAT_APPROVAL_REQUIRED');
+const CHAT_APPROVAL_DAYS = nconf.get('CHAT_APPROVAL_DAYS');
+
+let api = {};
```

Comment:
Instead of a new  lib I think we should keep using nconf.set like we already do for some config cars here https://github.com/HabitRPG/habitica/blob/develop/website/server/libs/setupNconf.js, since all these values are static